src/Shared/CommunicationsUtilities.cs

```diff
@@ -292,9 +293,8 @@ internal static void SetEnvironment(IDictionary<string, string> newEnvironment)
 
         /// <summary>
         /// Given a base handshake, generates the real handshake based on e.g. elevation level.  
-        /// Client handshake required for comparison purposes only.  Returns the update handshake.  
         /// </summary>
-        internal static long GenerateHostHandshakeFromBase(long baseHandshake, long clientHandshake)
+        internal static long GenerateHostHandshakeFromBase(long baseHandshake)
```

Comment:
Make private or inline, no need to expose it anymore.

Reply:
👍